src/main/java/org/mars/rover/kata/MarsRover.java

```diff
@@ -1,13 +1,14 @@
 package org.mars.rover.kata;
 
 import lombok.Getter;
-
+import lombok.Setter;
 
 @Getter
+@Setter
 public class MarsRover {
```

Comment:
A w sumie to dlaczego nie `public record MarsRover`? 😉

Reply:
Zostawiam class, ponieważ record powinien być niemutowalny, czyli nie można dać instancji jakiejś klasy wewnątrz :)